src/dbos.ts

```diff
@@ -121,6 +121,11 @@ function httpApiDec(verb: APITypes, url: string) {
   }
 }
 
+export interface StartWorkflowParams {
+  workflowID?: string;
+  queueName?: string;
```

Comment:
Maybe make it the entire queue object instead of just the name? Not a huge deal either way.

Reply:
There are reasons why not to use the whole object.  Internally, we will eventually be able to get the object by name anyway.